test/NuGet.Core.FuncTests/NuGet.Commands.FuncTest/RestoreCommandTests.cs

```diff
@@ -1852,6 +1852,8 @@ public async Task RestoreCommand_LockedLockFileWithOutOfDateProjectAsync()
             }
         }
 
+#if IS_DESKTOP
+        // TODO: To work on coreclr we need to address https://github.com/NuGet/Home/issues/7588
```

Comment:
👏 
We definitely need to prioritize this as soon as possible because we should not ship a regression, even in a preview,

Reply:
Feel free to jump on the issue: https://github.com/NuGet/Home/issues/7588